tests/console/check_system_info.pm

```diff
@@ -111,11 +113,10 @@ sub run {
     if (check_var('VERSION', get_required_var('UPGRADE_TARGET_VERSION'))) {
         check_milestone_version;
         my $myaddons = get_var('SCC_ADDONS', "");
-        $myaddons .= ",base,serverapp"                             if (is_sle('15+')                                   && check_var('SLE_PRODUCT', 'sles'));
-        $myaddons .= ",base,desktop,we,python2"                    if (is_sle('15+')                                   && check_var('SLE_PRODUCT', 'sled'));
-        $myaddons .= ",base,serverapp,desktop,dev,lgm,python2,wsm" if (is_sle('<15', get_var('ORIGIN_SYSTEM_VERSION')) && is_sle('15+'));
-        $myaddons .= ",python2"                                    if (is_sle('=15', get_var('ORIGIN_SYSTEM_VERSION')) && is_sle('15+'));
-        $myaddons .= ",base,serverapp,desktop,dev,lgm,python2,wsm,phub" if (is_leap_migration);
```

Comment:
@tinawang123 Do you cover the scenario which has mid step:

e.g.

15SP2-SP3 then SP4, if we remove this check, it may impact older releases.

Reply:
Won't test this scenario.